app/src/org/commcare/activities/UpdateActivity.java

```diff
@@ -247,6 +247,14 @@ protected void startUpdateCheck() {
         uiController.downloadingUiState();
     }
 
+    private void initUpdateTaskProgressDisplay() {
```

Comment:
In normal updates, the update check portion does not use a progress dialog, it just shows a progress bar embedded in the main activity view. But since I am showing a blank screen for consumer app updates, I want the generic consumer app progress dialog to show, so I need to control it manually.